### PR TITLE
Add support for filename when uploading media

### PR DESCRIPTION
**Pull request by w1r0x**
There is a problem with filename when uploading files

When you trying to save file through Riot app it saving it like this (without correct filename):
image

PR fix this issue by passing filename when uploading file

### CODE REVIEW

File: matrix_client/api.py [hunk 1]
```diff
@@ -744,12 +744,17 @@ def _send(self, method, path, content=None, query_params=None, headers=None,
 
         return response.json()
 
-    def media_upload(self, content, content_type):
+    def media_upload(self, content, content_type, filename=None):
+        query_params = dict()
```
Comment (Zil0) [inline matrix_client/api.py]: `query_params = {}` is the preferred way of doing this.

**Comment by Zil0** [comment]
Thanks, LGTM if not for the review comment :)

**Comment by non-Jedi** [comment]
LGTM once you signoff per Contributing.rst. Thanks!

**Comment by w1r0x** [comment]
Changes done.

**Pull request merged**